src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,59 @@
+.nav {
+  display: block;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__items {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+
+    &:link {
+      position: relative;
+      display: flex;
+      text-decoration: none;
+      color: $main-text-color;
+      text-transform: uppercase;
+      font-size: 12px;
+      font-weight: 500;
+      align-items: center;
+      cursor: pointer;
+
+      padding: 23px 0;
+    }
+
+    &:visited {
+      color: $main-text-color;
+    }
+
+    &:hover {
+      color: $button-color;
+    }
+  }
+}
+
+.is-active {
+  color: $button-color !important;
```

Comment:
Avoid "!important", it's a bad practice. To avoid this problem with specificity you can style just ".nav__link" instead of ".nav__link:link" so they will have the same specificity and then you can reorder them.